### ISSUE TITLE
Bug: Notice: Undefined index: REMOTE_ADDR in [/var/www/MISP/app/Model/AppModel.php, line 3916]

### ISSUE THREAD

**Issue by cudeso** [MISP/MISP#8795]
### Expected behavior

Changing MISP configuration settings via cake without a Notice error

### Actual behavior

When setting configuration changes via cake there is now always a PHP notice error.

```
sudo /var/www/MISP/app/Console/cake admin setSetting "MISP.baseurl" https://www.localhost.com
Notice Error: Undefined index: REMOTE_ADDR in [/var/www/MISP/app/Model/AppModel.php, line 3916]

2022-11-30 14:57:19 Notice: Undefined index: REMOTE_ADDR in [/var/www/MISP/app/Model/AppModel.php, line 3916]
/var/www/MISP/app/tmp/logs/[51090]: serverSettingsEdit -- Server setting changed -- "MISP.baseurl (https:\/\/192.168.1.1\/) => (https:\/\/www.localhost.com)"
Setting "MISP.baseurl" changed to "https://www.localhost.com"
```

This seems to be introduced by https://github.com/MISP/MISP/commit/192ed311b989f50af9ed784b5c4d58383dd0042e

### Steps to reproduce

From the console: `sudo /var/www/MISP/app/Console/cake admin setSetting "MISP.baseurl" https://www.localhost.com`

### Version

2.4.166

### Operating System

Ubuntu

### Operating System version

20.04

### PHP version

PHP 7.4.3

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by github-germ** [comment]
Thx. This is a dup of https://github.com/MISP/MISP/issues/8788

**Comment by JakubOnderka** [comment]
Fixed in #8757
